samples/sensor/mcux_acmp/src/main.c

```diff
@@ -16,7 +16,8 @@
 #define ACMP_POSITIVE 5
 #define ACMP_NEGATIVE 5
 #define ACMP_DAC_VREF 0
-#elif (defined(CONFIG_BOARD_MIMXRT1170_EVK_CM7) || defined(CONFIG_BOARD_MIMXRT1170_EVK_CM4))
+#elif (defined(CONFIG_BOARD_MIMXRT1170_EVK_MIMXRT1176_CM7) || \
+	defined(CONFIG_BOARD_MIMXRT1170_EVK_MIMXRT1176_CM4))
```

Comment:
if there's only one cpu for this then it can be shortened to the singular board name as mentioned above